### PR TITLE
Add emotion, event of SenseVoice.

### PR DESCRIPTION
**Pull request by fbzhong**
Returns the emotion, event of SenseVoice, as https://github.com/k2-fsa/sherpa-onnx/issues/1184

### CODE REVIEW

File: sherpa-onnx/csrc/offline-recognizer-sense-voice-impl.h [hunk 1]
```diff
@@ -52,6 +52,11 @@ static OfflineRecognitionResult ConvertSenseVoiceResult(
 
   r.words = std::move(src.words);
 
+  // parse lang, emotion and event from tokens.
+  r.lang = sym_table[src.tokens[0]];
```
Comment (csukuangfj) [inline sherpa-onnx/csrc/offline-recognizer-sense-voice-impl.h]: Could you wrap it inside an if statement:
```c++
if (src.tokens.size() >= 3) {
  r.lang = sym_table[src.tokens[0]];
  r.emotion = sym_table[src.tokens[1]];
  r.event = sym_table[src.tokens[2]];
}
```

If users provide a blank audio file, I am not sure what the model would return. But put it inside an if statement prevents accessing invalid elements when `src.tokens` is empty.

File: sherpa-onnx/kotlin-api/OfflineRecognizer.kt [hunk 2]
```diff
@@ -96,7 +99,10 @@ class OfflineRecognizer(
         val text = objArray[0] as String
         val tokens = objArray[1] as Array<String>
         val timestamps = objArray[2] as FloatArray
-        return OfflineRecognizerResult(text = text, tokens = tokens, timestamps = timestamps)
+        val lang = objArray[3] as String
+        val emotion = objArray[4] as String
+        val event = objArray[5] as String
+        return OfflineRecognizerResult(text = text, tokens = tokens, timestamps = timestamps, lang = lang, emotion = emotion, event = event)
```
Comment (csukuangfj) [inline sherpa-onnx/kotlin-api/OfflineRecognizer.kt:105]: Could you please also update the java API at
https://github.com/k2-fsa/sherpa-onnx/blob/f300ec0f985904e577c0155e3aa87a7e57d58f70/sherpa-onnx/java-api/src/com/k2fsa/sherpa/onnx/OfflineRecognizerResult.java#L6-L9
?

**Review by csukuangfj — COMMENTED** [review]
Thanks! Just left some minor comments. Otherwise, it looks great!

**Comment by fbzhong** [comment]
Fix all those issues.

**Comment by csukuangfj** [comment]
Thank you for your first-time contribution!

**Pull request merged**